genesis-tools/genesis-populate/src/main.rs

```diff
@@ -14,7 +14,7 @@ fn main() {
         .arg(
             Arg::with_name("home")
                 .long("home")
-                .default_value(&default_home)
+                .default_value(default_home.to_str().unwrap())
```

Comment:
```suggestion
                .default_value_os(default_home.as_os_str())
```

Path are not utf8, but all good Rust API handle that. There should (almost) never be a need to call `to_str`

Reply:
fixed here and in other places